components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -1589,6 +1589,15 @@ private static void buildSAMLAuthenticationConfiguration(IdentityProvider fedIdp
         property.setValue(paramMap.get("logoutUrl"));
         properties.add(property);
 
+        property = new Property();
+        property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.IS_SLO_REQUEST_ACCEPTED);
+        if ("on".equals(paramMap.get("sloRequestAccepted"))) {
+            property.setValue("true");
+        } else {
+            property.setValue("false");
+        }
```

Comment:
Lets move these to constants

Reply:
Fixed with PR https://github.com/wso2/carbon-identity-framework/pull/3738